app/public/js/controllers/PatientsListCtrl.js

```diff
@@ -1,67 +1,36 @@
-angular.module('PatientsListCtrl', []).controller('PatientsListController',
-function($compile, $location, $scope, DTOptionsBuilder, DTColumnBuilder) {
-
-	$scope.title = 'État des patients';
-	$scope.dtInstance = {};
-	$scope.dtOptions = DTOptionsBuilder.newOptions()
-		.withOption('data', [
-			{id: 1, picture: null, firstName: 'Roy', lastName: 'Benjamin', age: 78, sex: 'Homme', isStable: true},
-			{id: 2, picture: null, firstName: 'Tran', lastName: 'Austin-Didier', age: 78, sex: 'Femme', isStable: false},
-		])
-		.withPaginationType('full_numbers')
-		.withOption('rowCallback', rowCallback)
-		.withOption('createdRow', createdRow)
-		.withBootstrap();
-
-	$scope.dtColumns = [
-		DTColumnBuilder.newColumn('id').withTitle('ID'),
-		DTColumnBuilder.newColumn('picture').withTitle('Photo').renderWith(setProfilPicture),
-		DTColumnBuilder.newColumn('firstName').withTitle('Prénom'),
-		DTColumnBuilder.newColumn('lastName').withTitle('Nom'),
-		DTColumnBuilder.newColumn('age').withTitle('Âge'),
-		DTColumnBuilder.newColumn('sex').withTitle('Sexe'),
-		//DTColumnBuilder.newColumn(null).withTitle('Actions').notSortable().renderWith(actionsHtml)
-	];
-
-	$scope.addPatient = function() { console.log('Add patient'); }
-
-	function createdRow(row, data, dataIndex) {
-		// Recompiling so we can bind Angular directive
-		if (!data.isStable) {
-			$('td', row).css('background-color', '#FFCDD2');
+var app = angular.module('PatientsListCtrl', []);
+
+app.controller('PatientsListController',
+	function($location, $scope, DTOptionsBuilder, DTColumnDefBuilder, firebaseService) {
+
+		$scope.title = 'État des patients';
+		$scope.patients = firebaseService.getAll('patients');
```

Comment:
I think this should be specified at some point that it is a promise. Either make the getAll method manage promises, or specify it so another coder can use it to their advantages.